### DISCUSSION TITLE
blog/dimensionality-reduction-auto-encoders

### DISCUSSION THREAD

**Discussion by giscus** [PrabhuKiran8790/codedojo#3] [General]
# blog/dimensionality-reduction-auto-encoders

Let's look at how we can perform the dimensionality reduction using the Auto-Encoders using Tensorflow.

https://codedojo.vercel.app/blog/dimensionality-reduction-auto-encoders

**Comment by Shayaan-Hussain** [comment]
Hello Testing Comments

**Reply by Shayaan-Hussain** [reply]
Testing Replies

**Reply by Shayaan-Hussain** [reply]
Well...Seems like it's working great. Good job
